README.md

```diff
@@ -201,6 +201,43 @@ $ cd [OUTPUT_DIR]  # make a separate directory for each workflow
 $ caper run [WDL] -i [INPUT_JSON]
 ```
 
+
+## Caper server heartbeat (running multiple servers)
+
+Caper server writes a heartbeat file (specified by `--server-heartbeat-file`) on every 120 seconds (controlled by `--server-heartbeat-timeout`). This file has simply an IP(hostname)/PORT pair of the running `caper server`.
```

Comment:
Minor re-wording suggestions: "Caper server writes a heartbeat file (specified by `--server-heartbeat-file`) every 120 seconds (controlled by `--server-heartbeat-timeout`). This file will contain an IP(hostname)/PORT pair of the running `caper server`.

Reply:
Done